test/duration_test.exs

```diff
@@ -111,5 +122,23 @@ defmodule DurationTests do
     assert d.megaseconds + d.seconds + d.microseconds > 0
   end
 
+  describe "now" do
+    test "now/0 returns the duration since epoch" do
+      erlang_now_microseconds = div(:os.system_time, 1000)
+      timex_now_microseconds = Timex.Duration.now() |> Timex.Duration.to_microseconds
+      difference_microseconds = timex_now_microseconds - erlang_now_microseconds
+      assert difference_microseconds < 1000000
+      assert difference_microseconds >= 0
+    end
```

Comment:
Do you think this test does a good enough job of asserting that a Duration struct is returned by `now/0`?